gtda/homology/simplicial.py

```diff
@@ -5,6 +5,7 @@
 from types import FunctionType
 
 import numpy as np
+from gph import ripser_parallel as ripser
```

Comment:
We should here think how to support the parameter `n_threads` from `gph`. But I do not think this is the right PR to think about that.

Already using `gph` should provide by itself a faster execution even in single thread mode.

Reply:
Correct, I agree!  This PR is just a minimal one, as you have already figured out.